mjpc/tasks/humanoid/tracking/task.cc

```diff
@@ -37,6 +38,55 @@ std::tuple<int, int, double, double> ComputeInterpolationValues(double index,
 
   return {index_0, index_1, weight_0, weight_1};
 }
+
+// return length of motion trajectory
+int TrajectoryLength(int id) {
+  // Jump
+  if (id == 0) {
```

Comment:
Can we comment this also with the the mocap filename ids (e.g. `CMU-CMU-88-88_09_poses.xml`).

Reply:
Done.